src/mme-app/mmeStates/attachStates.cpp

```diff
@@ -191,8 +191,8 @@ void AttachWfImsiValidateAction::initialize()
         }
         {
                 ActionTable actionTable;
-                actionTable.addAction(&ActionHandlers::send_identity_request_to_ue);
-                actionTable.setNextState(AttachWfIdentityResponse::Instance());
+                actionTable.addAction(&ActionHandlers::send_identification_request_to_old_mme);
```

Comment:
What if tmsi does not map to old Mme then should we not send identity request to UE ?

Reply:
We are planning to implement the negative scenarios after the positive scenario code runs successfully.